model/host/db.go

```diff
@@ -487,6 +487,12 @@ func FindStaleRunningTasks(cutoff time.Duration) ([]task.Task, error) {
 			},
 		},
 	})
+	pipeline = append(pipeline, bson.M{
+		"$project": bson.M{
+			task.IdKey:        1,
+			task.ExecutionKey: 1,
+		},
+	})
```

Comment:
why not merge this into the above declaration? why do we need to do appens at all?

Reply:
I find that this is easier to read